src/main/utils/cmds/npm.ts

```diff
@@ -46,7 +47,7 @@ export class NpmCmd {
     }
 
     protected static getNpmLsArgs(projectRoot: string): string {
-        let args: string = `${Flag.Json} ${Flag.All}`;
+        let args: string = `${Flag.Json} ${Flag.All} ${Flag.Silent}`;
```

Comment:
Based on my understanding, the current implementation will not function as intended because an error is thrown by 
```ScanUtils.executeCmd('npm ls' + this.getNpmLsArgs(projectRoot), projectRoot)```. Consequently, the subsequent `JSON.parse` operation cannot be executed.

To address this issue, it seems necessary to capture the output of the command and parse it as JSON in order to access the dependencies section from npm ls.

Reply:
Yes, it is that, and JSON is on stdout while logs are on stderr.

Reply:
```
JSON is on stdout while logs are on stderr.
```
Right but we still need something to read the stdout.